c_cxx/fns_candy_style_transfer/fns_candy_style_transfer.c

```diff
@@ -10,7 +9,10 @@
 #include "providers.h"
 #endif
 #include <objbase.h>
+#else
+#include <png.h>
```

Comment:
do we need to include png.h here or is it enough to do so from image_file_libpng.c?